staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -375,9 +383,9 @@ func (r *crdHandler) updateCustomResourceDefinition(oldObj, newObj interface{})
 	if !apiextensions.IsCRDConditionTrue(newCRD, apiextensions.Established) &&
 		apiextensions.IsCRDConditionTrue(newCRD, apiextensions.NamesAccepted) {
 		if r.masterCount > 1 {
-			r.establishingController.QueueCRD(newCRD.Name, 5*time.Second)
+			r.establishingController.QueueCRD(newCRD.Tenant+"/"+newCRD.Name, 5*time.Second)
```

Comment:
nit: format +"/"+

Reply:
Fixed.

Looks like a bug in hack/update-gofmt.sh. I ran "make update" but it was missed.